contents/newsletter/think-like-a-growth-engineer.md

```diff
@@ -0,0 +1,111 @@
+
+Their work comes down to tracking and improving platform or business-level metrics. For example, our growth team cares about new revenue, expansion, signup conversion, activation, retention, and more. Their planning, prioritization, and execution are focused on improving these metrics.
+
+The tradeoff is caring less about products, features, and user requests. They aren't planning their roadmap months in advance or caring about accurate project timelines. They let the metrics be their north star and do whatever, wherever to improve it.
+
+> **Takeaway:** Figure out the status quo for the area you work on. To do this:
+> 1. Define and track your activation and retention numbers. 
+> 2. Track how they move week over week. 
+> 3. Track how they move with the changes you ship.
+>
+> You can't think like a growth engineer if you don't have a metrics baseline to start from.
+>
+> For more advanced users, create a dashboard using a framework like [AARRR](/product-engineers/aarrr-pirate-funnel) or [growth loops](/product-engineers/growth-loops) to monitor your key flow conversions, activation, and retention.
+> ![AARRR and growth loops](https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2024_09_17_at_11_56_15_193aeaa117.png)
+
+## 2. Develop an experimentation mindset
+
+The big way growth engineers improve the metrics they care about is through experiments. Doing this well requires developing the elusive "experimentation mindset." This differs from the traditional software engineering mindset in a few ways:
+
+1. **Hypothesis over requirements.** Software engineers work on features users clearly need. Growth engineers work on more unknowns and care more about exploration and discovery. To do this, they develop hypotheses and run experiments to validate their assumptions.
+
+2. **Iteration over stability.** While most engineers focus on developing stable, bug-free code, growth engineers would rather fail fast and iterate. For example, a software engineer might feel uncomfortable adding a dependency before evaluating it, while a growth engineer will ship it to help them test faster. 
+
+3. **Pragmatism over perfection.** Growth engineers know their experiments might fail and get removed. This means they ship the "good enough" version over the maintainable and scalable one. They know they can always improve it later.
+
+> **Takeaway:** Instead of blindly shipping the next feature on your list, think about the hypothesis behind it.
+>
+> For example, adding customizable notifications will get users to return more often, improving retention. After adding the feature, use the metrics you set up in the previous section to validate your hypothesis.
+
+## 3. Prioritize like a growth engineer
+
+Engineers are like bus drivers, helping move a product along its roadmap toward success. Growth engineers are like Uber drivers, bouncing from experiment to experiment stacking little wins that payoff in the long term. 
```

Comment:
Just realized this would make a great visual – could we get Lottie do a couple of hogs, one driving a bus on a fixed route, another an uber? Nice to have, not must have.